lv_conf_template.h

```diff
@@ -215,6 +215,9 @@
 
     /** Enable drawing complex gradients in software: linear at an angle, radial or conical */
     #define LV_USE_DRAW_SW_COMPLEX_GRADIENTS    0
+
+    /* Enable registering of custom SW draw handlers */
+    #define LV_DRAW_SW_CUSTOM_BLEND_HANDLERS  0
```

Comment:
IMO this config is not required as enabling this feature by default doesn't make a big difference.

Reply:
Yeah, it's really a trade off. On the flip side (besides the more complex testing) I also see the issue of more config options. In this case the extra memory usage is only a few bytes which is less of an issue compared to the testing and config complexity increase. 

So unless there is a principal objection, please remove this config.